tests/driver_epd_bw_spi_disp_dev/main.c

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2019 Silke Hofstra
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the black/white e-paper SPI display driver.
+ *
+ * @author      Silke Hofstra <silke@slxh.eu>
+ * @}
+ */
+
+
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "ztimer.h"
+
+#include "disp_dev.h"
+
+#include "epd_bw_spi.h"
+#include "epd_bw_spi_disp_dev.h"
+#include "epd_bw_spi_params.h"
+
+#include "pictures.h"
+
+const uint8_t riot_32_width = 32;
+const uint16_t riot_32_height = sizeof(riot_logo_32) / sizeof(riot_logo_32[0]);
+const uint8_t riot_200_width = 200;
+const uint8_t riot_200_height = 200;
+
+int main(void)
+{
+    epd_bw_spi_t epd;
+
+    int init = epd_bw_spi_init(&epd, epd_bw_spi_params);
+
+    if (init != 0) {
+        printf("EPD_BW_SPI INIT FAILED: %i\n", init);
+        return 1;
+    }
+
+    disp_dev_t *dev = (disp_dev_t *)&epd;
+    disp_dev_area_t large_area = {
+        .x1 = 0,
+        .x2 = riot_200_width - 1,
+        .y1 = 0,
+        .y2 = riot_200_width - 1.
+    };
+
+    dev->driver = &epd_bw_spi_disp_dev_driver;
+
+    while (1) {
+        /* Set both RAM buffers to the RIOT logo */
+        disp_dev_map(dev, &large_area, (uint16_t *)riot_icon_200);
+        disp_dev_map(dev, &large_area, (uint16_t *)riot_icon_200);
+
+        /* Draw small RIOT logos on the display */
+        for (int y = 0; y < disp_dev_width(dev); y += riot_32_height) {
+            for (int x = 0; x < disp_dev_height(dev); x += 32) {
```

Comment:
```suggestion
        for (unsigned y = 0; y < disp_dev_width(dev); y += riot_32_height) {
            for (unsigned x = 0; x < disp_dev_height(dev); x += 32) {
```

Or maybe even `uint16_t`